importer/Uploader.py

```diff
@@ -46,11 +46,12 @@ def add_descriptions(self, target_item, descriptions, log):
             desc_content = descriptions[description]['value']
             lang = descriptions[description]['language']
             self.wdstuff.add_description(
-                lang, desc_content, target_item, summary=self.summary)
+                lang, desc_content, target_item)
             if log:
                 t_id = target_item.getID()
-                log.logit(t_id + " ADDED DESCRIPTION " +
-                          lang + " " + desc_content)
+                message = "{} ADDED DESCRIPTION {} {}".format(
+                    t_id, lang, desc_content)
+                log.logit(message)
 
     def add_aliases(self, target_item, aliases, log):
```

Comment:
Related to this. Here you access `editAliases` directly which means that again the edit summary  does not get added. You should be able to just use `self.wdstuff.addLabelOrAlias` again (assuming add_labels, has been run before).

Or are there situations where you would on purpose want an alias without having a label?